src/utils/Processing.ts

```diff
@@ -240,6 +241,46 @@ export function fillTank(rawFSHes: RawFSH[], config: Configuration): FSHTank {
   return new FSHTank(docs, config);
 }
 
+export function checkNullValuesOnArray(resource: any, parentName = '', priorPath = ''): void {
+  const resourceName = parentName ? parentName : resource.id ?? resource.name;
+  for (const propertyKey in resource) {
+    const property = resource[propertyKey];
+    const currentPath = !priorPath ? propertyKey : priorPath.concat(`.${propertyKey}`);
+    // If a property's key begins with "_", we'll want to ignore null values on it's top level
+    // but still check any nested objects for null values
+    if (propertyKey.startsWith('_')) {
+      if (isPlainObject(property))
+        // If we encounter an object property, we'll want to check its properties as well
+        checkNullValuesOnArray(property, resourceName, currentPath);
```

Comment:
This is a fairly small code-styling question, but I think we tend to use `{}` around `if` statements that are on more than one line. I'm not 100% if that's truly a pattern we follow or just something I tend to do normally. Do you know if this is a pattern or do you have a preference for keeping the `{}` off?

Reply:
Yeah, I think it is a bit of an unwritten rule -- and I think I assumed eslint or prettier would enforce it -- but we pretty much always use `{` `}` except in one-liners.

Reply:
This is definitely weird! Initially the if-statement was a one-liner, but prettier separated the clause out and I just ran with it. This has been fixed in the most recent commit.